src/frontend/src/lib/components/hero/HeroContent.svelte

```diff
@@ -71,12 +71,12 @@
 	{/if}
 
 	{#if usdTotal}
-		<div transition:slide={SLIDE_PARAMS}>
+		<div in:slide={SLIDE_PARAMS}>
 			<ExchangeBalance />
 		</div>
 	{/if}
 
-	<div transition:slide|local={SLIDE_PARAMS} class="flex w-full justify-center text-left">
+	<div in:slide|local={SLIDE_PARAMS} class="flex w-full justify-center text-left">
```

Comment:
not for this PR, but I introduced the `|local` when I introduced the transition of the parent `Hero` too. Not really sure, but maybe it can be removed

Reply:
Good point! We can have a look afterwards indeed